pkg/entrypoint/run.go

```diff
@@ -0,0 +1,295 @@
+/*
+Copyright 2018 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package entrypoint
+
+import (
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"os/signal"
+	"path/filepath"
+	"strconv"
+	"syscall"
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+const (
+	// InternalErrorCode is what we write to the marker file to
+	// indicate that we failed to start the wrapped command
+	InternalErrorCode = 127
+	// AbortedErrorCode is what we write to the marker file to
+	// indicate that we were terminated via a signal.
+	AbortedErrorCode = 130
+
+	// DefaultTimeout is the default timeout for the test
+	// process before SIGINT is sent
+	DefaultTimeout = 120 * time.Minute
```

Comment:
This feels like a holdover from the Prow code, which we should drop. Our tasks can run for longer than 2h, and I don't think we have a good way to pipe _our_ default timeout (10m) here, so we should just drop it.

Reply:
this is removed now.  The timeout that can be set in the Build Spec is handled by Build killing the entire pod, is that correct?  Is that why that setting doesn't need to be propogated here?